feat: scratch org resume (#551)
forcedotcom/sfdx-core: messages/scratchOrgCreate.md

# WaitValidationMaxError

Expected 'wait' greater than or equal to %s but received %s.

# NoScratchOrgInfoError

> No ScratchOrgInfo was found. Check that the ID and the DevHub are correct

Author: it's the Id of the ScratchOrgInfo record, which will be used as `--job-id` for the resume

# ScratchOrgDeletedError

> That scratch org has been deleted and you can't connect to it

Reviewer: ```suggestion
That scratch org has been deleted, so you can't connect to it anymore.
```

# StillInProgressError

> The scratch org is not ready yet (Status = %)

Reviewer: ```suggestion
The scratch org is not ready yet (Status = %). 
```

# action.StillInProgress

> Wait and try the <%= config.bin %> <%= command.id %> command again

Reviewer: ```suggestion
Wait and try the <%= config.bin %> <%= command.id %> command again. 
```

Reviewer: Question: Can we add a bit more info about how long to wait?  Yeah, I know, it's impossible to know.  But it would be good to give some idea, such as wait for a few minutes or hours or days, you know?  For example "Wait for a few minutes, and then try the ....".

# WaitValidationMaxError

Expected 'wait' greater than or equal to %s but received %s.

> # NoScratchOrgInfoError

Reviewer 2: SF style guide asks that error message entries carry an `error.` prefix

Author: oh, for libraries, too?  That doesn't seem to be the case anywhere else for core v3.

Reviewer 2: Have a look at my core PR :). @jshackell-sfdc does the message prefixes only apply to plugins?

Author: @mdonnalley all the errors in core v3 should be renamed?

Reviewer 3: I don't think libraries need to adhere to the plugin style guide. 

Personally I'd prefer to keep all the error names within sfdx-core consistent
